### PR TITLE
changed 'no contract' button using bootstrap class

### PR DESCRIPTION
**Pull request by Vishakha7906**
Targets #{2354}

### Description
no contract button changes using bootstarp class

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/Project/Resources/views/index.blade.php [hunk 1]
```diff
@@ -83,10 +83,10 @@
                                         </span>
                                     @endforeach
                                 </td>
-                                <td>
+                                <td>  <div class="text-center">
+                                
                                     @if(empty($project->projectContracts->first()->contract_file_path))
-                                        <span class="badge badge-light border border-dark rounded-0">No Contract</span>
-                                    @endif
+                                    <div class="text-center"> <span class="badge badge-pill badge-secondary">No Contract</span> </div>                                    @endif
```
Comment (Ayush8923) [inline Modules/Project/Resources/views/index.blade.php:89]: Why have you used extra `div` here?
We don't need extra `div` here.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comments.

File: Modules/Project/Resources/views/index.blade.php [hunk 2]
```diff
@@ -83,10 +83,10 @@
                                         </span>
                                     @endforeach
                                 </td>
-                                <td>
+                                <td>  <div class="text-center">
```
Comment (Ayush8923) [inline Modules/Project/Resources/views/index.blade.php:86]: @Vishakha7906 Why did you use extra div here? Whereas we don't need this extra div. And as far as I see this extra div is not closed.

File: package-lock.json [hunk 3]
```diff
@@ -4,7 +4,6 @@
     "requires": true,
     "packages": {
```
Comment (Ayush8923) [inline package-lock.json:5]: Please remove this `package.json` file from your PR. We don't need this file.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Please check the comments.

File: Modules/Project/Resources/views/index.blade.php (hunk 2 above)
Comment (Vishakha7906) [reply Modules/Project/Resources/views/index.blade.php:86]: @Ayush8923 I have done the required changes can you please review it again.

**Comment by gautam071** [comment]
@Vishakha7906 Please fix the conflicts

**Comment by Vishakha7906** [comment]
@gautam071 I am facing some issues with my laptop so that's why it is taking time.
I hope I will complete it by tomorrow.

**Pull request closed without merging**